experimental-default-theme.json

```diff
@@ -0,0 +1,23 @@
+{
+	"global": {
+		"color": {
+			"primary": "#52accc",
+			"background": "white",
+			"text": "black"
+		}
+	},
+	"features": {
+		"typography": {
+			"dropCapEnabled": false
+		}
+	},
+	"blocks": {
```

Comment:
I'm thinking that we probably want to group this theme.json _per feature_ to ease extraction (styles, features, etc) and each key would have a "global" and a "blocks" subkey.

Example:

```
{
    "styles": {
        "global": {
            "color": { ...}
        },
        "blocks": {
            "core/paragraph": {
                "color": { ... }
            }
        }
    },
    "features": { // or config, editor, or something like that
        "global": { ... },
        "blocks": {
            "core/paragraph": { ... }
        }
    }
}
```

For context, [the shape](https://github.com/WordPress/gutenberg/issues/21583#issuecomment-613911005) Riad originally proposed, to which I [suggested](https://github.com/WordPress/gutenberg/pull/22076#issuecomment-624183378) using "styles" for the things relative to the global styles.

Reply:
I think I'd like even more something with less hierarchy: 

```
{
    "styles": {
        "global": {
            "color": { ...}
        },
        "core/paragraph": {
            "color": { ... }
        }
    },
    "features": { // or config, editor, or something like that
        "global": { ... },
        "core/paragraph": { ... }
    }
}
```

This way, when we extract the data in `useEditorFeature`, it's as easy as:

```js
has( getSettings(), [
    '__experimentalConfig', // we only need one key for this
    context, // 'global' or the block name
    feature,
] )
```

Reply:
@nosolosw, great idea to group it like this when passing to the block editor as part of settings object. How would you structure the `theme.json` to optimize for the person that needs to maintain it? 😄 I think those two not necessarily need to align.

Reply:
@nosolosw, I think you answered in https://github.com/WordPress/gutenberg/pull/22291#discussion_r425107659 :) Le's fold this discussion and continue there.